src/seedu/addressbook/commands/ListCommand.java

```diff
@@ -11,15 +12,35 @@
 public class ListCommand extends Command {
 
     public static final String COMMAND_WORD = "list";
+    public static final int PAGE_CONTAIN = 3;
 
     public static final String MESSAGE_USAGE = COMMAND_WORD
-            + ": Displays all persons in the address book as a list with index numbers.\n"
-            + "Example: " + COMMAND_WORD;
+            + ": Displays all persons or persons in one page in the address book as a list with index numbers.\n"
+            + "Example: " + COMMAND_WORD + "\n"
+            + "Example: " + COMMAND_WORD + " 1";
 
+    public ListCommand(int targetPage) { super(targetPage); }
+    public ListCommand() {}
```

Comment:
as mentioned and explained in tutorial, do try to avoid the use of single line methods.